website/docs/getting-started.mdx

```diff
@@ -0,0 +1,455 @@
+
+1. Install the flux CLI
+
+```
+brew install fluxcd/tap/flux
+```
+
+For For other installation methods, see the relevant [Flux documentation](https://fluxcd.io/docs/installation/#install-the-flux-cli).
+
+2. Export your credentials
+
+```
+export GITHUB_TOKEN=<your-token>
+export GITHUB_USER=<your-username>
+```
+
+3. Check your Kubernetes cluster
+
+```
+flux check --pre
+```
+
+The output is similar to:
+```
+► checking prerequisites
+✔ kubernetes 1.22.2 >=1.20.6
+✔ prerequisites checks passed
+```
+
+4. Install Flux onto your cluster with the `flux bootstrap` command
+
+```
+flux bootstrap github \
+  --owner=$GITHUB_USER \
+  --repository=fleet-infra \
+  --branch=main \
+  --path=./clusters/my-cluster \
+  --personal
+```
+
+Full installation documentation including how to work with other Git providers is available [here](https://fluxcd.io/docs/installation/).
+
+The bootstrap command above does following:
+
+- Creates a git repository fleet-infra on your GitHub account
+- Adds Flux component manifests to the repository
+- Deploys Flux Components to your Kubernetes Cluster
+- Configures Flux components to track the path /clusters/my-cluster/ in the repository
+
+### Configure access to the dashboard
+
+For this guide we will use the cluster user, for complete documentation including how to configure an OIDC provider see the documentation [here](./configuration/securing-access-to-the-dashboard.mdx).
+
+We will generate a bcyrpt hash for your chosen password and store as a secret in Kubernetes. There are several different ways to generate a bcrypt hash, this guide uses an Alpine Docker image to generate one.
+
+1. Run an Alpine Docker image interactively and supply the password of your choice as an environment variable by changing `<your-password>`:
+
+```
+docker run -e CLEAR_PASSWORD="<your-password>" -it alpine
+```
+
+2. Once inside the shell environment of the Alpine image, install the bcrypt library dependencies as well as the bcrypt library itself:
+
+```
+apk add --update musl-dev gcc libffi-dev python3 python3-dev py3-pip
+pip install bcrypt
+```
+
+3. Run the following Python script to generate a hash:
+
+```
+python3 -c 'import bcrypt, os; print(bcrypt.hashpw(os.getenv("CLEAR_PASSWORD").encode(), bcrypt.gensalt()))'
+```
+
+you should get a response like:
+
+```
+b'$2b$12$FFokfwM/p8RPr0YMTdHWk.MXe3Kg1/UVyj63pT9FU8/mthUjKjm7W'
+```
+
+4. Copy the output from the above command and exit the shell
+
+```
+exit
+```
+
+5. Now create a Kubernetes secret to store your chosen username and the password hash.
+
+Note when you update the password that you only need content between ' marks.
+
+```
+kubectl create secret generic cluster-user-auth \
+  --namespace flux-system \
+  --from-literal=username=admin \
+  --from-literal=password='$2b$12$FFokfwM/p8RPr0YMTdHWk.MXe3Kg1/UVyj63pT9FU8/mthUjKjm7W'
+```
+
+6. Configure RBAC for the admin user
+
+Still in your local clone of `fleet-infra`, create a new YAML file and add the following:
```

Comment:
It's only in the next section we're told to clone `fleet-infra`.

Unlike the next section, this doesn't tell you to put the file in `clusters/my-cluster`